src/routes/tickets/util/getMetadataFromTicket.ts

```diff
@@ -1,11 +1,29 @@
 import { ITicket, ITicketMetaData } from '../controllers/ticketController';
 
+
 export const getMetadataFromTicket = (data: ITicket[]): ITicketMetaData[] => {
```

Comment:
Why is there an array of tickets passed in if you are getting the metadata for a specific ticket

Reply:
True, don't need to do that. I'll fix

Reply:
Because the callback object sets the "previewText" on the main object and not in the messages array

Reply:
?!

Reply:
Let me clarify :)

the object hat is returned by the function alters the main object from the Ticket i.e the previewText.

messages only exists on the Ticket object but the previewText will not exist in the messages object but rather the ticket object which then becomes the metadata ticket object

Reply:
Wait nvm

Reply:
My new solution worked first try 🥳 
![external-content duckduckgo com](https://user-images.githubusercontent.com/16852656/146834201-3df4ec96-4a4d-4b38-8475-35a1f86961c5.gif)